src/main/java/com/rackspacecloud/metrics/queryservice/configuration/SecurityProperties.java

```diff
@@ -1,11 +1,21 @@
 package com.rackspacecloud.metrics.queryservice.configuration;
 
+import lombok.Getter;
+import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
 
-@Configuration
+import javax.validation.constraints.NotEmpty;
+
+@Component
 @ConfigurationProperties(prefix = "security")
 public class SecurityProperties {
```

Comment:
FYI adding lombok's `@Data` here saves having to declare getter/setter on each. Also declares some other handy bits.